File: Web/src/test/java/form/HelpForm.java
package form;
import aquality.selenium.elements.interfaces.IButton;
import aquality.selenium.forms.Form;
import org.openqa.selenium.By;

public class HelpForm extends Form {

    private final IButton buttonSend = getElementFactory().getButton(By.xpath("//button[contains(@class,'help-form__send-to-bottom-button')]"), "send to button");

    public HelpForm() {
        super(By.xpath("//h2[contains(@class,'help-form__title')]"), "Help form");
    }

    public void clickButtonSend() {
        buttonSend.click();
    }
}

File: Web/UserInterfaceTest/UserInterfaceTest/Forms/Pages/HelpForm.cs
using OpenQA.Selenium;
using Aquality.Selenium.Elements.Interfaces;


namespace UserInterfaceTest.Forms.Pages
{
    public class HelpForm : BaseAppForm
    {
        private IButton SendToBottom => ElementFactory.GetButton(By.XPath("//button[contains(@class,'send-to-bottom')]"),"Send to bottom button");
        public HelpForm() : base(By.XPath("//h2[contains(@class,'form__title')]"),"Help form header") 
        {
        }

        public void ClickSendToBottom() => SendToBottom.Click();
    }
}
